src/ui/ui.css

```diff
@@ -19,52 +19,112 @@ body {
+  background-size: 100%;
+  width: 11664px;
+  height: 36px;
+  -webkit-animation: mspin-medium-film 5332ms infinite steps(324);
+  -moz-animation: mspin-medium-film 5332ms infinite steps(324);
+  animation: mspin-medium-film 5332ms infinite steps(324);
+}
+
+@-webkit-keyframes mspin-medium-film {
```

Comment:
No need for vendor-prefixed `@-webkit-keyframes` either - just leave the main one.

Reply:
Done